druid-shell/src/backend/windows/clipboard.rs

```diff
@@ -18,20 +18,26 @@ use std::ffi::CString;
 use std::mem;
 use std::ptr;
 
+use winapi::shared::minwindef::MAX_PATH;
 use winapi::shared::minwindef::{FALSE, UINT};
 use winapi::shared::ntdef::{CHAR, HANDLE, LPWSTR, WCHAR};
 use winapi::shared::winerror::ERROR_SUCCESS;
 use winapi::um::errhandlingapi::GetLastError;
+use winapi::um::shellapi::DragQueryFileW;
 use winapi::um::winbase::{GlobalAlloc, GlobalLock, GlobalSize, GlobalUnlock, GMEM_MOVEABLE};
+use winapi::um::winuser::CF_HDROP;
 use winapi::um::winuser::{
     CloseClipboard, EmptyClipboard, EnumClipboardFormats, GetClipboardData,
     GetClipboardFormatNameA, IsClipboardFormatAvailable, OpenClipboard, RegisterClipboardFormatA,
     SetClipboardData, CF_UNICODETEXT,
 };
+use winapi::um::shellapi::{HDROP};
 
-use super::util::{FromWide, ToWide};
 use crate::clipboard::{ClipboardFormat, FormatId};
 
+use super::util::ToWide;
+use super::util::FromWide;
+
```

Comment:
Let's revert this change. The old `use super::util::{FromWide, ToWide};` is better.